examples/BUILD

```diff
@@ -40,4 +40,5 @@ bazel_integration_test(
 
 bazel_integration_test(
     name = "bzlmod_example",
+    override_bazel_version = "6.0.0-pre.20221012.2",
```

Comment:
there's a 6.0.0-rc0 this morning, maybe use that?

Reply:
That release tag doesn't appear to be available on GitHub. I think the one I'm referencing here is the latest.

Reply:
it should be 6.0.0-rc1

Reply:
sorry, "6.0.0rc1" without the dash

Reply:
oops sorry I was too fast and inaccurate :)